tests/trilinos/trilinos_sparsity_pattern_03.cc

```diff
@@ -39,7 +39,7 @@ void test ()
   columns.add_range(3*myid, 3*myid+3);
   columns.compress();
 
-  sp.reinit(rows, columns, MPI_COMM_WORLD, 0);
+  sp.reinit(rows, columns, MPI_COMM_WORLD, (dealii::types::global_dof_index)0);
```

Comment:
Yea, this is damn ugly. How does this work for everyone other than you? What's different on your system and what is the error you get?